src/Resolve/graphtype.jl

```diff
@@ -570,6 +570,26 @@ function wipe_snapshots!(graph::Graph)
     return graph
 end
 
+
+function color_string(c, str...)
+    return sprint((io, args) -> printstyled(io, args...; color=c), str, context=stderr)
+end
+
+# system colors excluding greys, bright-yellow and dark-blue
+const CONFLICT_COLORS = [5,1,6,3,2,13,12,9,10];
+const pkgID_color = let  # Cycle through each color in turn when a new color is needed
```

Comment:
Inconsistent capitalization with e.g. the global variable just above it. It could also go inside the `let`?

Reply:
This could also just be put inside the `Graph` struct? No need for it to be global?